piou-piou/ribs-cron-bundle/1.0/config/ribs_cron.yaml

```diff
@@ -0,0 +1,6 @@
+parameters:
+    data_directory: '%kernel.project_dir%/data/'
+    ribs_cron_ip_external:
+    ribs_cron_ip_internal: "127.0.0.1"
```

Comment:
But you should still use environment variables here. Not the plain string value.

Reply:
I have made the changes